drivers/usb/device/usb_dc_b9x.c

```diff
@@ -878,8 +878,12 @@ static int usb_irq_init(void)
 
 	usbhw_enable_manual_interrupt(FLD_CTRL_EP_AUTO_CFG | FLD_CTRL_EP_AUTO_DESC |
 				FLD_CTRL_EP_AUTO_FEAT | FLD_CTRL_EP_AUTO_STD);
+	usbhw_set_eps_irq_mask(FLD_USB_EDP5_IRQ | FLD_USB_EDP6_IRQ);
+
 	core_interrupt_enable();
```

Comment:
Do we really need to enable global ISRs. It's driver code so kernel is running in that case ISR's are already enabled. Please check all same entries in this file.

Reply:
@andriy-bilynskyy 
Do you mean line 881?

Reply:
@andriy-bilynskyy I removed core_interrupt_enable(), it works ok